deploy/helm/values.yaml

```diff
@@ -1,6 +1,11 @@
 # Default values for security-profiles-operator.
 replicaCount: 3
 
+
+spoimage:
+  repository: gcr.io/k8s-staging-sp-operator/security-profiles-operator
```

Comment:
In order to better integrate with automatic dependency update tools, I'd suggest splitting the repository and registry keys as shown in the following example: https://docs.renovatebot.com/modules/manager/helm-values/

This would look as follows:

```yaml
spoImage:
  registry: gcr.io
  repository: k8s-staging-sp-operator/security-profiles-operator
  tag: latest
```

In case someone would set up an explicit image tag in their values file, this would then be automatically picked up and receive updates. It's also consistent to what other helm charts are doing in general.